### DISCUSSION TITLE
13 days and no coins

### DISCUSSION THREAD

**Discussion by VIPJeKa** [blockscout/blockscout#9928] [Support]
Sent 1.893995355711487945 milkALGO and they don’t come
Transaction Hash - 0x9acca3b80e53e448109370fc9762092f885580f84dfbc69e83fb87fce6e492c2
![Screenshot_11](https://github.com/blockscout/blockscout/assets/167015493/5f51ca71-22c4-472e-a464-a301eff21a2b)
![Screenshot_12](https://github.com/blockscout/blockscout/assets/167015493/0ebd9e48-86d6-4521-9420-3d6a79765fe0)

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
